Dockerfile

```diff
@@ -29,10 +38,14 @@ RUN apk --no-cache add --virtual build-dependencies gcc g++ musl-dev go \
     && apk del --purge -r build-dependencies \
     && rm -rf ~/go
 
-COPY docker-config.json /home/jenkins/.docker/config.json
```

Comment:
If removing this, perhaps we should remove the entire block that installs amazon-ecr-credential-helper